mechanism/http/src/main/java/org/wildfly/security/mechanism/http/_private/ElytronMessages.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.security.mechanism.http._private;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Log messages and exceptions for Elytron.
+ *
+ * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ */
+@MessageLogger(projectCode = "ELY", length = 5)
+public interface ElytronMessages extends BasicLogger {
```

Comment:
This can be changed to package private and moved up to the ```http``` directory (i.e., no need for the``` _private``` package here).

Reply:
The same thing can also be done for the ```org.wildfly.security.http._private.ElytronMessages``` class now that the ```impl``` directory is gone.

Reply:
fixed.